### ISSUE TITLE
Should we have a separate documentation page on accessibility and examples of how to calculate different measures ?

### ISSUE THREAD

**Issue by christophfink** [r5py/r5py#323]

**Comment by HTenkanen** [comment]
+1 I think this is a good idea! 👍🏻

**Comment by christophfink** [comment]
This should include #327

**Comment by wklumpen** [comment]
Agreed. I have some short write-ups I've done several times from a more mathematical perspective (and general one) based on some of the more theoretical papers for a course. Attached is the PDF
[CIV1599_F22_Handouts.pdf](https://github.com/r5py/r5py/files/12346949/CIV1599_F22_Handouts.pdf)

Can expand that into a broader example.

**Comment by wklumpen** [comment]
Most important thing of all: Let's not call a measure that isn't inverse square a gravity measure :smile:
